pkg/assets/static/redirectpage.js

```diff
@@ -0,0 +1,189 @@
+const configURL = "/api/v1/authconfig"; // URL to get config
+const queryString = window.location.search;
+const urlParams = new URLSearchParams(queryString);
+const link = urlParams.get("link");
+const keyword = urlParams.get("keyword");
+const selectedId = urlParams.get("selectedId");
+const consoleUrlMock = "https://console.redhat.com/openshift/sandbox";
```

Comment:
Instead of hardcoding this consleURL (and why it's called consoleUrl**Mock** btw?) let's load it from the config (`/api/v1/authconfig`)
So let's maybe do the following:
- rename `consoleUrlMock` to something like `signupURL`.
- try to load it from `configURL` JSON as `signup-url`.
- if there is no `signup-url` defined in the config payload JSON then fall back to the default value (which is just the base URL: `/`)
- We, the dev sandbox team, will add `https://console.redhat.com/openshift/sandbox` to the config API for our production sandbox.

Reply:
> We, the dev sandbox team, will add `https://console.redhat.com/openshift/sandbox` to the config API for our production sandbox.

See https://github.com/codeready-toolchain/registration-service/pull/464